src/filternew.jl

```diff
@@ -0,0 +1,32 @@
+#ideia:
+# dado de entrada da tabela: somente as colunas com os ativos a serem analisados
+
+function estima_coeficientes(tabela::DataFrame; num_linha::Integer = 40, M::Integer = 10^5, K::Integer = 5, ativoj::Integer = 1) 
```

Comment:
Se eu colocasse nessa função uma tabela com menos de 40 linhas, o que ia acontecer? 
Pensa aí como generalizar seu código pra não dar esse erro. Ou seja, se for mais de 40 linhas ele corta em 40, se forem menos de 40 linhas ele deixa o numero de linhas da tabela